app/models/user.rb

```diff
@@ -1,12 +1,16 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
Please see the below error which occurs when the user clicks on the `log out` button.
![sign_out](https://user-images.githubusercontent.com/35504149/158605770-93905ead-cac3-43f2-8f51-da8f2595f008.png)

Reply:
Hello dear @SajjadAhmad14 , I think for this milestone, implementing the ` log out` button was not among the project requirements as you can see [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md). I add it just for future uses but let me implement it.
thank you for your highlighting review!

best regards!